tests/gtests/test_eltwise.cpp

```diff
@@ -393,6 +409,13 @@ class eltwise_test_t : public ::testing::TestWithParam<eltwise_test_params_t> {
                 && tag != memory::format_tag::aBcde8b
                 && tag != memory::format_tag::aBcde16b);
     }
+    bool hip_check_format_tag(memory::format_tag tag) {
+        // Blocking is not supported by MIOpen
+        return (tag != memory::format_tag::aBcd8b
+                && tag != memory::format_tag::aBcd16b
+                && tag != memory::format_tag::aBcde8b
+                && tag != memory::format_tag::aBcde16b);
+    }
```

Comment:
```suggestion
    bool hip_check_format_tag(memory::format_tag tag) {
        // HIP has the same limitations for `tag` as CUDA.
        return cuda_check_format_tag(tag);
    }
```

Reply:
Addressed the above change of utilizing  cuda_check_format_tag for hip_check_format_tag